packages/llvm.rb

```diff
@@ -1,276 +1,104 @@
 require 'package'
 
 class Llvm < Package
-  description 'The LLVM Project is a collection of modular and reusable compiler and toolchain technologies. The optional packages clang, lld, lldb, polly, compiler-rt, libcxx, libcxxabi and openmp are included.'
+  description 'The LLVM Project is a collection of modular and reusable compiler and toolchain technologies. The optional packages clang, lld, lldb, polly, compiler-rt, libcxx, libcxxabi, and openmp are included.'
   homepage 'http://llvm.org/'
-  version '10.0.0'
+  @_ver = '11.1.0'
+  version @_ver + '-rc1'
```

Comment:
`version @_ver`?

Reply:
I'll do that as soon as the 11.1.0 is officially tagged? As it is the two patches to bring it to 11.1.0 have been sitting there for the last 8 days tagged as 11.1.0-rc1 with no new commits.

Reply:
OK, then rename the source and binaries appropriately.  We should accurately display the correct version.

Reply:
Sounds reasonable.

Reply:
Uploaded.